heka/os_telemetry_collector.sls

```diff
@@ -0,0 +1,10 @@
+{%- if pillar.heka.os_telemetry_collector is defined %}
+
+include:
+- heka._common
+
```

Comment:
you need to add this : 
```
{%- from "heka/map.jinja" import os_telemetry_collector with context %}
{%- set server = os_telemetry_collector %}
```
https://github.com/tcpcloud/salt-formula-heka/blob/stacklight/heka/metric_collector.sls#L6-L7